subsys/testsuite/coverage/coverage.h

```diff
@@ -117,4 +117,12 @@ struct gcov_info {
 
 };
 
+/**
```

Comment:
Just so this isn't interpreted as Doxygen (that can be done later, if needed)
```suggestion
/*
```